app/Decsys/Repositories/Contracts/IWordlistRepository.cs

```diff
@@ -6,4 +6,5 @@ public interface IWordlistRepository
 {
     UserWordlist List(string ownerId);
     Task<UserWordlist> Create(string ownerId);
+    Task PutRule(string wordlistId, int ruleIndex, WordlistRules ruled);
```

Comment:
```suggestion
    Task PutRule(string wordlistId, int ruleIndex, WordlistRules rule);
```
I think just a mistype? (also keeps it inline with your mongo implementation) sorry kind of annoying I know, but feels important to be clear on the interface as WordListRules is plural, but this rule is singular

Reply:
Good spot, pretty sure the type should be singular